source/user-manual/user-administration/index.rst

```diff
@@ -12,7 +12,7 @@ In the password management section, you can find instructions on how to use the
 
 The RBAC section contains directions on how to create Wazuh indexer users, also known as internal users, assign them different roles and map them to the Wazuh manager API. Find out how to create an admin user, a read-only user, a custom user, and a user with permission to read and manage only a group of agents. 
 
-In the single sign-on section, you can find instructions on how to integrate Wazuh with different Identity Providers to implement Single Sign-On. Find instructions for Okta, Azure Active Directory, PingOne, Google, Jumpcloud and OneLogin. 
+In the single sign-on section, you can find instructions on how to integrate Wazuh with different Identity Providers to implement Single Sign-On. Find instructions for Okta, Microsoft Entra ID, PingOne, Google, Jumpcloud and OneLogin. 
 
 In the LDAP integration section, you can find instructions on how to integrate Wazuh with LDAP/Active Directory to authenticate and authorize users.
```

Comment:
Please consider the following "Active Directory" occurrences, if applicable:

- Up here, where it says "_LDAP/**Active Directory** to authenticate and authorize users_".
- In RBAC `/user-manual/user-administration/rbac` section.
- In the new LDAP `/user-manual/user-administration/ldap` section.
-  In `/user-manual/user-administration/single-sign-on/` sections. Please consider a url change and redirection for the `azure-active-directory` page.